tailwind.config.js

```diff
@@ -42,12 +42,18 @@ module.exports = {
 
       animation: {
         'fade-in': 'fade-in 100ms ease',
+        tilt: 'tilt 10s infinite linear',
       },
       keyframes: {
         'fade-in': {
           '0%': { opacity: 0 },
           '100%': { opacity: 1 },
         },
+        tilt: {
+          '0%, 50%, 100%': { transform: 'rotate(0deg)' },
+          '25%': { transform: 'rotate(1.5deg)' },
+          '75%': { transform: 'rotate(-1.5deg)' },
+        },
```

Comment:
The glow effect is actually a div that sits behind the progress component with a blur effect applied to it. It isn't a box shadow or a border.

This tilt effect creates an optical illusion of subtle movement on the glow.